localisation/ko/docs/00-setup.md

```diff
@@ -0,0 +1,351 @@
+
+1. VS Code 버전을 확인하세요.
+
+    ```bash
+    code --version
+    ```
+
+   `1.99.0` 이상이 권장됩니다. 더 낮은 버전이라면 [Visual Studio Code 설치 페이지](https://code.visualstudio.com/)를 방문하여 지침을 따르세요.
+
+   > **참고**: `code` 명령을 실행할 수 없을 수 있습니다. 이 경우 설정을 위해 [이 문서](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)를 따르세요.
+
+### Visual Studio Code 시작
+
+1. 작업 디렉터리를 생성하세요.
+1. 다음 명령을 실행하여 이 저장소를 포크하고 로컬 머신에 복제하세요.
+
+    ```bash
+    gh repo fork microsoft/github-copilot-vibe-coding-workshop --clone
+    ```
+
+1. 복제된 디렉터리로 이동하세요.
+
+    ```bash
+    cd github-copilot-vibe-coding-workshop
+    ```
+
+1. 터미널에서 VS Code를 실행하세요.
+
+    ```bash
+    code .
+    ```
+
+1. VS Code 내에서 새 터미널을 열고 다음 명령을 실행하여 저장소 상태를 확인하세요.
+
+    ```bash
+    git remote -v
+    ```
+
+   다음을 확인할 수 있어야 합니다. `origin`에 `microsoft`가 보이면 포크된 저장소에서 다시 복제해야 합니다.
+
+    ```bash
+    origin  https://github.com/<your GitHub ID>/github-copilot-vibe-coding-workshop.git (fetch)
+    origin  https://github.com/<your GitHub ID>/github-copilot-vibe-coding-workshop.git (push)
+    upstream        https://github.com/microsoft/github-copilot-vibe-coding-workshop.git (fetch)
+    upstream        https://github.com/microsoft/github-copilot-vibe-coding-workshop.git (push)
+    ```
+
+1. 두 확장 프로그램이 모두 설치되어 있는지 확인하세요 &ndash; [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot)과 [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat).
+
+    ```bash
+    # Bash/Zsh
+    code --list-extensions | grep github.copilot
+    ```
+
+    ```powershell
+    # PowerShell
+    code --list-extensions | Select-String "github.copilot"
+    ```
+
+   아무것도 보이지 않으면 해당 확장 프로그램이 아직 설치되지 않았다는 의미입니다. 다음 명령을 실행하여 확장 프로그램을 설치하세요.
+
+    ```bash
+    code --install-extension "github.copilot" --force && code --install-extension "github.copilot-chat" --force
+    ```
+
+### MCP 서버 설정
+
+1. `$REPOSITORY_ROOT` 환경 변수를 설정하세요.
+
+   ```bash
+   # bash/zsh
+   REPOSITORY_ROOT=$(git rev-parse --show-toplevel)
+   ```
+
+   ```powershell
+   # PowerShell
+   $REPOSITORY_ROOT = git rev-parse --show-toplevel
+   ```
+
+1. MCP 서버 설정을 복사하세요.
+
+    ```bash
+    # bash/zsh
+    cp -r $REPOSITORY_ROOT/docs/.vscode/. \
+          $REPOSITORY_ROOT/.vscode/
+    ```
+
+    ```powershell
+    # PowerShell
+    Copy-Item -Path $REPOSITORY_ROOT/docs/.vscode/* `
+              -Destination $REPOSITORY_ROOT/.vscode/ -Recurse -Force
+    ```
+
+1. F1을 입력하거나 Windows에서 `Ctrl`+`Shift`+`P`, Mac OS에서 `Cmd`+`Shift`+`P`를 눌러 명령 팔레트를 열고 `MCP: List Servers`를 검색하세요.
+1. `context7`을 선택한 다음 `Start Server`를 클릭하세요.
+
+## GitHub Copilot 에이전트 모드 확인
+
+1. GitHub Codespace 또는 VS Code 상단의 GitHub Copilot 아이콘을 클릭하고 GitHub Copilot 창을 여세요.
+
+   ![Open GitHub Copilot Chat](./images/setup-02.png)
+
+1. 로그인 또는 가입을 요청받으면 진행하세요. 무료입니다.
+1. GitHub Copilot 에이전트 모드를 사용하고 있는지 확인하세요.
+
+   ![GitHub Copilot Agent Mode](./images/setup-03.png)
```

Comment:
This image path should point to the original one, which is `../../../docs/images/setup-03.png`.